src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/Router.kt

```diff
@@ -21,7 +22,11 @@ class Router(val config: RouteConfig)
         val urls: MutableList<String> = mutableListOf()
     }
 
-    private fun transform(x: Any) = Serializer.instance.toResult(x)
+    private fun transform(x: Any) = when (x)
+    {
+        is AuthenticationResponse -> Serializer.instance.gson.toJson(x)!!
```

Comment:
The response for a login should conform to the standard protocol, so we don't want to cast the response to our custom `Result` model in this case.